### ISSUE TITLE
Disabling the extension does not restore title bars

### ISSUE THREAD

**Issue by rrthomas** [material-shell/material-shell#263]
**Describe the bug**
When I disable the extension, windows I had open are still missing their title bars.

**Expected behavior**
When disabling the extension, title bars should be restored.

**To Reproduce**
- Occurs on:
  - [x] X.Org
- GNOME Shell version: 3.36
- Distro: Ubuntu 20.04
- Steps to reproduce, if applicable:
1. Install/enable the extension
2. Disable the extension.
4. Note that title bars are not restored.

**Current commit**
Version 3 from extensions.gnome.org

**Comment by PapyElGringo** [comment]
Thanks for the feedbacks

**Comment by rrthomas** [comment]
You're welcome! Good luck with this project, it's exciting to see efforts to bring tiling WMs to GNOME finally.

**Comment by Crucizer** [comment]
I'm having the same problem, can you please suggest how to fix this?

**Comment by Crucizer** [comment]
Here's the fix for this problem: `gsettings set org.gnome.desktop.wm.preferences button-layout :minimize,maximize,close`
Restart the gnome shell and the problem will be fixed but I recommend the owner to do by default.

**Comment by lfom** [comment]
> Here's the fix for this problem: `gsettings set org.gnome.desktop.wm.preferences button-layout :minimize,maximize,close`
> Restart the gnome shell and the problem will be fixed but I recommend the owner to do by default.

I don't think it would interfere with MS. There was a bug in the code that caused a crash when disabling the extension that could cause this, but it was fixed some time ago. No more problems disabling/enabling the extension, either on XOrg or Wayland.

**Comment by sebastiansam55** [comment]
This is still a thing, my maximize and minimize buttons were gone after trying out the extension for a few minutes.

Added them back via gnome tweaks (equivalent to the `gsettings` command I believe.
